### ISSUE TITLE
CMake bad generation when cpp_info.libs contains multiple entries

### ISSUE THREAD

**Issue by packadal** [conan-io/conan#850]
Hi,

I am generating a package for openjpeg, which contains both openjp2 and openjp3d libs.

When generating with `self.cpp_info.libs = ["openjp2", "openjp3d"]` the lib is not found by `conanbuildinfo.cmake`.

This is due to the following line in `conanbuildinfo.cmake` :
```
    conan_find_libraries_abs_path(${CONAN_LIBS_OPENJPEG} ${CONAN_LIB_DIRS_OPENJPEG}
                                  CONAN_FULLPATH_LIBS_OPENJPEG)
```
When `${CONAN_LIBS_OPENJPEG}` contains multiple entries, the list is expanded, and the second element of the list is used for `${CONAN_LIB_DIRS_OPENJPEG}`, which screws up the `find_library` later.

This is easily fixed by changing the line to:

```
    conan_find_libraries_abs_path("${CONAN_LIBS_OPENJPEG}" ${CONAN_LIB_DIRS_OPENJPEG}
                                  CONAN_FULLPATH_LIBS_OPENJPEG)
```

**Comment by sixten-hilborn** [comment]
This has been fixed in the development branch:  #840
We just have to wait for the next release :)

**Comment by packadal** [comment]
Awesome ! I did not see that, I guess I'll close this issue then :)

**Comment by memsharded** [comment]
Thanks all for reporting!
This fix has been released in 0.18.1, please update and re-open and report if necessary.
